python-squid/src/main/java/org/sonar/python/api/tree/Tree.java

```diff
@@ -42,6 +42,8 @@
 
   List<Tree> children();
 
+  List<Token> tokens();
```

Comment:
I would not add this method to `Tree`, only to `PyStatementListTree`, at least for the moment.